#PID参数
#for xy
Ki: 0
Kd: 0
Kp: 5.0

# for z
Ki_: 0
Kd_: 0
Kp_: 1.2


#Output limits (pid->out)
limMin: -1.0
limMax: 1.0


limMin_: -1.0
limMax_: 1.0
#Integrator limits
limMinInt: -10.0
limMaxInt: 10.0

limMinInt_: -10.0
limMaxInt_: 10.0
# Sample time
T: 0.05
#用于检测小车停止
limvel: 2.0
#takeof vel 
takeof_vel: 0.2
#系数
coff: 0.0026